tmt/steps/prepare/install.py

```diff
@@ -239,10 +247,36 @@ def go(self, guest):
 
             # Use both dnf install/reinstall to get all packages refreshed
             # FIXME Simplify this once BZ#1831022 is fixed/implemeted.
-            guest.execute(f"{command} install {options} {rpms_directory}/*")
             if 'rpm-ostree' not in command:
+                guest.execute(f"{command} install {options} {rpms_directory}/*")
                 guest.execute(f"{command} reinstall {options} {rpms_directory}/*")
-            summary = fmf.utils.listed(local_packages, 'local package')
+                local_packages_installed.append(local_packages)
+            else:
+                for package in local_packages:
+                    check = f'dnf list installed {os.path.basename(package)}'
+                    if 'x86_64' in check:
+                        check = check.strip('x86_64.rpm')
+                    elif 'aarch64' in check:
+                        check = check.strip('aarch64.rpm')
+                    else:
+                        self.debug(f"Local rpm package arch not supported", level=3)
```

Comment:
Why is it necessary to check for specific `arch` here? What about `noarch` packages?

Reply:
Right now, automotive only supports these two architectures, further, I couldn't find a reliable method to match and strip the arch from the package name for the install command. 
But in this I guess I've excluded noarch packages now, any suggestions? I see you have imported re already, just couldn't come up with a reliable pattern

Reply:
as far as a separate method, so have one check for rpm-ostree in command and call a new method? Then I'd have to add an additional method for the "standard" path, and refactor the go method i think, with a lot of duplicated code.

Please correct me here in my understanding

Reply:
Casually following along here, so take my suggestion with a grain of salt...

You can inspect the arch of an RPM package like so:

```
$ rpm -qp --queryformat='%{ARCH}\n' rpm-ostree-2021.1-3.fc33.x86_64.rpm 
x86_64
```

It would be another step in the code, but would be more reliable if you need to check the arch

Reply:
Just to note, the reason I am keeping the already installed detection is because, rpm-ostree returns in error when it finds the packages already installed.
Of course i could simplify here and just keep the inside try except, and keep the general error capture of "not installed" Let me know and I could eliminate some additional lines
the reason for the separate loop on local packages is because when using the wildcard, for rpm-ostree, if one package is found already installed the loop stops, and there will not be an attempt to install all packages in the dir.
Maybe this behavior is a bug...